src/Hangfire.Tags.PostgreSql/Hangfire.Tags.PostgreSql.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Authors>Yong Liu</Authors>
+    <Company>Medidata Solutions Inc</Company>
+    <Version>1.0.0</Version>
+    <Description>Support for PostgreSql for Hangfire.Tags. This seperate library is required in order to search for tags, and proper cleanup.</Description>
+    <Copyright />
+    <PackageProjectUrl>https://github.com/face-it/Hangfire.Tags</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/face-it/Hangfire.Tags/blob/master/LICENSE.md</PackageLicenseUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/face-it/Hangfire.Tags/master/Icon.png</PackageIconUrl>
+    <RepositoryUrl>https://github.com/face-it/Hangfire.Tags</RepositoryUrl>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>hangfire tags PostgreSql</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Hangfire.Tags.PostgreSql.csproj.DotSettings" />
```

Comment:
not needed; .csproj.DotSettings isn't in the github repo neither

Reply:
okay